_rules/table-header-cells-refer-to-data-cells-d0f69e.md

```diff
@@ -27,28 +27,21 @@ acknowledgments:
 
 ## Applicability
 
-The rule applies to any HTML element that
+The rule applies to any HTML element for which all of the following is true:
 
-- has the [semantic role][] of [rowheader][] or [columnheader][] and;
-- is [visible][] and;
-- is [included in the accessibility tree][] and;
-- has at least one ancestor in the [flat tree][] with a [semantic role][] of either [table][] or [grid][]; and
-- whose closest ancestor in the [flat tree][] with a [semantic role][] of either [table][] or [grid][] is [included in the accessibility tree][].
+- The element has the [semantic role][] of [rowheader][] or [columnheader][]; and
```

Comment:
I think we should put this in the sentence above, instead of as part of the list. This isn't some condition, headers is what this rule is about.